bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="siemenshvac" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd ">
+
+
+	<type>binding</type>
+	<name>SiemensHvac Binding</name>
+	<description>This is the binding for SiemensHvac.</description>
+	<connection>local</connection>
+
+	<discovery-methods>
+		<discovery-method>
+			<service-type>upnp</service-type>
+			<match-properties>
+
+				<match-property>
+					<name>modelName</name>
+					<regex>Web Server OZW672.01</regex>
```

Comment:
In the code you have:
```java
if (modelName.startsWith("Web Server OZW672.01")) {
```

So I think this needs to be:
```suggestion
					<regex>Web Server OZW672.01.*</regex>
```

Where you able to test this suggestion finder feature? I can provide some tips if needed.

Reply:
I change the regex.
I don't find a way to test this so far. I don't know if discovery service is working in development environment?

Reply:
While openHAB is not running, you can edit runtime/etc/addons.xml and add insert the contents of your addon.xml file. Then clean cache and start openHAB. Your binding should then be suggested when you go to Add-on Store:

![image](https://github.com/openhab/openhab-addons/assets/19519842/6d2d9292-378c-4f2e-a072-b42f25b23b7d)

If it does not work, you can have a closer look at what's going on by:
```shell
log:set trace org.openhab.core.config.discovery.addon
log:set trace org.openhab.core.config.discovery.addon.upnp
```

Reply:
Thanks for your guidelines.
With this, I was able to verify that it works ok.

![image](https://github.com/openhab/openhab-addons/assets/4678815/c1e5ae6f-58b7-4902-b971-cd767ae62823)